BUILD

```diff
@@ -1,36 +1,23 @@
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
 
-exports_files(
-    [
-        "versions/master/docs/bazel-user-manual.html",
-        "command-line-reference-prefix.html",
-        "command-line-reference-suffix.html",
-    ],
-)
-
-DOT_GRAPH_HTML_FILES = [
-    "versions/master/docs/query.html",
-    "versions/master/docs/build-ref.html",
-]
-
-filegroup(
-    name = "srcs",
-    srcs = glob(["**"]),
-    visibility = ["//:__pkg__"],
-)
-
 filegroup(
     name = "jekyll-srcs",
     srcs = glob(
         ["**/*"],
         exclude = [
+            ".git/**",
+            "bazel-*/**",
             "BUILD",
+            "WORKSPACE",
             "jekyll-tree.sh",
+            "scripts/**",
             "*.swp",
-            "command-line-reference-prefix.html",
-            "command-line-reference-suffix.html",
             "site/README.md",
-        ] + DOT_GRAPH_HTML_FILES,
+            "LICENSE",
+            "CONTRIBUTING",
+            "production/**",
+            "README.md",
+        ],
     ),
```

Comment:
To keep bazel-release you can add specifically the file bazel-release, instead of globbing it.